### ISSUE TITLE
dplay.se - ERROR: Geoblocked video

### ISSUE THREAD

**Issue by Sopor-** [spaam/svtplay-dl#719]
When i try to download from dplay.se i get this error. I have tried a few others and then it works. 
```
$ svtplay-dl -P hls http://www.dplay.se/varldenstuffastejobb/season-1-broarbetare-fargtillverkare-industrisanerare/
ERROR: No videos found.
ERROR: Geoblocked video

```
Someone else that get the same error when downloading any episode of varldenstuffastejobb?

**Comment by granbom** [comment]
The above URL works for me.

```
$ svtplay-dl -P hls http://www.dplay.se/varldenstuffastejobb/season-1-broarbetare-fargtillverkare-industrisanerare/
INFO: Selected to download hls, bitrate: 6327
INFO: Outfile: varldens.tuffaste.jobb.med.carina.berg.s01e08.broarbetare.fargtillverkare.industrisanerare-213556-dplay.ts
```

**Comment by Sopor-** [comment]
I'm still using the same IP address i did for several month ago. I believe it must almost a year now.
I get the same error on my other computer that have a different public IP address.
I can view the episode through Retrospect without any geoblocking.
I'm running svtplay-dl 1.9.6-23-gad94873. What version do you run @granbom ?

**Comment by Sopor-** [comment]
Yes, 1.9.6 is working but not latest master 1.9.6-23-gad94873

**Comment by granbom** [comment]
It has.
This line will fix the key error that ad94873402b5809e3be265ed0df4e0b67b31b818 tried to fix:
`cookie = {"dsc-geo": quote('{{"countryCode":"{0}","expiry":{1}}}'.format(geo, timestamp))}`
Double curly-braces to fix the key error (since single ones messes up the format function).

The string returned by the str function have single quotes, so quote() encode them as %27. Dunno if that is the real issue, but that is the only difference I could find btw the strings.
Anyway, the above line works.
